src/Stryker.Core/Stryker.Core.UnitTest/Reporters/ConsoleReportReporterTests.cs

```diff
@@ -189,8 +189,7 @@ public void ConsoleReportReporter_ShouldPrintYellowOn80PercentAndLower()
                 Type = MutatorType.Arithmetic
             };
 
-            var target = new ConsoleReportReporter(new StrykerOptions("", "ReportOnly", "", 1000, null, "debug", false, false, 1, 80, 60, 0), chalkMock.Object);
-
+            var target = new ConsoleReportReporter(new StrykerOptions(thresholdLow:75), chalkMock.Object);
```

Comment:
thresholdLow should not be set to 75 here as the test name explicitly says the value should be 80. Since 80 is also the default value for threshold low, I suggest you do not pass any values to strykeroptions in this test and change the name to `ConsoleReportReporter_ShouldPrintYellowBetweenThresholdLowAndThresholdBreak`